src/NuGet.Services.AzureSearch/AzureSearchTelemetryService.cs

```diff
@@ -346,6 +346,27 @@ public void TrackDownloadSetComparison(int oldCount, int newCount, int changeCou
                 });
         }
 
+        public void TrackReadLatestIndexedExcludedPackagesList(int packageIdCount, TimeSpan elapsed)
```

Comment:
>TrackReadLatestIndexedExcludedPackagesList [](start = 20, length = 42)

This should be called in AuxiliaryFileClient right?

Reply:
actually not needed. will get rid of these. AuxiliaryFileClient tracks the auxiliary file downloads separately. This was before I was independently downloading aux file.